### PR TITLE
Fixed mobile notifications' styles

### PR DESCRIPTION
**Pull request by bconfortin**
#9419

### CODE REVIEW

File: packages/augur-ui/src/modules/account/components/notifications.tsx [hunk 1]
```diff
@@ -368,7 +374,7 @@ const Notifications = ({ toggle }: NotificationsProps) => {
       toggle={toggle}
       customClass={classNames({
         [Styles.DarkBackgroundMobile]: notificationCount !== 0,
-        [Styles.NoBorderAndBgOnMobile]: notificationCount === 0,
+        [Styles.NoBorderAndBgOnMobile]: notificationCount !== 0,
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/components/notifications.tsx]: if these are both going to get applied at the same time every-time (same condition to apply it) then why don't we just drop one of these classes and combine their definitions?

**Review by JohnDanz — CHANGES_REQUESTED** [review]

File: packages/augur-ui/src/modules/account/components/notifications.tsx (hunk 1 above)
Comment (bconfortin) [reply packages/augur-ui/src/modules/account/components/notifications.tsx]: Good one. Done.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**